src/textual/widgets/_input.py

```diff
@@ -236,6 +241,8 @@ def __init__(
             suggester: [`Suggester`][textual.suggester.Suggester] associated with this
                 input instance.
             validators: An iterable of validators that the Input value will be checked against.
+            validate_on: When does input validation happen? The default is to validate
```

Comment:
A rhetorical question doesn't make a good docstring. Suggest something like "zero or more of the following values... which define when to perform validation"